okhttp-tests/src/test/java/okhttp3/HeadersTest.java

```diff
@@ -811,4 +815,14 @@
         .build()
         .byteCount());
   }
+
+  @Test public void withDate() {
+    Date lowerBound = new Date();
```

Comment:
wanna just hardcode a date? Like `new Date(0)` and confirm it’s `1970-01-01T00:00:00Z`

Reply:
The builder would have to take a WallClock or some other time provider to do that in the current setup. I'll add an overload for withDate(Date date) so the test can use a constant date.